app/page.tsx

```diff
@@ -1,49 +1,33 @@
 import Link from 'next/link'
-import fs from 'fs'
+import Navigation from './components/navigation/navigation'
+import IPage from './models/IPage';
+import ISubpage from './models/ISubpage';
+import { getPages } from '@/app/utils/pages';
 
 export default function Home() {
-
-    const currentDirFilesList = fs.readdirSync(__dirname)
-    let archiveDirFilesList: string[] = []
-
-    if (currentDirFilesList.includes("archive")) {
-        console.log("Yay there is an 'archive' folder and it's contents are:")
-        archiveDirFilesList = fs.readdirSync(__dirname+"/archive")
-        console.log(archiveDirFilesList)
-    }
-
+    const pages = getPages() || [];
```

Comment:
fft: this seems like a simple enough refactor that could have been pulled into a separate PR and not get wrapped up in all the bigger more important changes you are doing. 😅